Casks/d/devonthink.rb

```diff
@@ -8,13 +8,21 @@
     end
   end
   on_big_sur :or_newer do
-    version "3.9.8"
-    sha256 "c56169fa98b72c2f043f6c2ca55c996a8fddc539e0df2b351ea45714e2427c1e"
+    version "3.9.9"
+    sha256 "718bb6dd403e93a77b5902f243da5f0103fbc5d7962949df3a8b6ba28cdf368e"
 
+    # The cask sometimes proposes a beta version, so we
+    # need to use a regex to ignore those.
     livecheck do
       url "https://api.devontechnologies.com/1/apps/sparkle/sparkle.php?id=300900000"
-      strategy :sparkle do |items|
-        items.map(&:version)
+      regex(/^v?(\d+(?:\.\d+)+)$/i)
+      strategy :sparkle do |items, regex|
+        items.map do |item|
+          match = item.version.match(regex)
+          next if match.blank?
+
+          match.to_s
+        end
```

Comment:
```suggestion
        items.map { |item| item.version[regex, 1] }
```
How about this? I've seen it in some other casks and it seems to work, but I'm unsure what the best practices for cask livechecks are. Perhaps @samford may know better.

Reply:
Yep, `item.version[regex, 1]` makes sense to me 👍